client/src/App.js

```diff
@@ -2,9 +2,15 @@ import { useState, useEffect } from 'react';
 import Day from './components/day';
 import TeacherAbsences from './page/teacherAbsences';
 import {BrowserRouter as Router, Routes, Route} from 'react-router-dom';
+import Dropdown from './components/dropdown';
+import Header from './components/header';
+
 function App() {
 
   return (
+   <div>
+    <Header/> 
```

Comment:
Not 100% necessary but running `Format Document` (Shift+Alt+F is VSCode) before committing helps with readability/consistent indentation